core/src/test/scala/org/broadinstitute/dsde/rawls/billing/GoogleBillingProjectLifecycleSpec.scala

```diff
@@ -119,7 +139,20 @@ class GoogleBillingProjectLifecycleSpec extends AnyFlatSpec {
         ArgumentMatchers.eq(SamBillingProjectPolicyNames.owner)
       )
     ).thenReturn(Future.successful(Map(WorkbenchEmail(userInfo.userEmail.value) -> Seq())))
-    val gbp = new GoogleBillingProjectLifecycle(mock[BillingRepository], samDAO, mock[GoogleServicesDAO])
+    val gbp = new GoogleBillingProjectLifecycle(repo, bpm, samDAO, mock[GoogleServicesDAO])
+
+    when(repo.setBillingProfileId(createRequest.projectName, profileModel.getId))
+      .thenReturn(Future.successful(1))
+
+    when(
+      bpm.createBillingProfile(
+        ArgumentMatchers.eq(createRequest.projectName.value),
```

Comment:
Since we've directly imported `ArgumentMatchers.any` it would be nice to be consistent and directly import other `ArgumentMatchers` too (a bit easier to read, IMO).

Reply:
I actually prefer the more explicit form, so will change the usage of `any` to be consistent.